### PR TITLE
Fix Bitbucket source url

### PR DESCRIPTION
**Pull request by carlosdevpereira**
@qinezh  Bitbucket repository url's use "src" instead of "blob" :)

### CODE REVIEW

**Review by qinezh — COMMENTED** [review]
Thanks your contribution @carlosdevpereira ,but seems it not works as expected, have you tested it?

File: src/linkInfo.ts [hunk 1]
```diff
@@ -13,11 +13,12 @@ export class LinkInfo {
     public toLink(): string {
         var startIndex = this.position.start.line + 1;
         var endIndex = this.position.end.line + 1;
-        var prefix = `https://${this.siteName}/${this.userName}/${this.repoName}/blob/${this.branchName}/${this.filePath}`;
+        var siteUrlSource = (this.siteName.split("@")[1] === "bitbucket.org") ? 'src' : 'blob';
```
Comment (qinezh) [inline src/linkInfo.ts:16]: why need `split("@")` here?

Comment (carlosdevpereira) [reply src/linkInfo.ts:16]: I debugged the extension within vscode and siteName was returning something like myUserName@siteDomain, (where myUserName is your git username and siteDomain is the remote domain), and to compare if the remote URL was from bitbucket or from github i splitted siteName to compare only the domain...

Comment (carlosdevpereira) [reply src/linkInfo.ts:16]: basically for bitbucket repositories siteName returns myUserName@siteDomain, and for github repositories it only returns siteDomain, either way, i did this way to work on both.. :)

Comment (qinezh) [reply src/linkInfo.ts:16]: Got it. I'll merge it and release a new version around these days. Thanks~

**Comment by qinezh** [comment]
@carlosdevpereira the new version of GitLink has been released.

**Comment by carlosdevpereira** [comment]
@qinezh Ok thanks. 👍

**Pull request merged**